certified-connectors/Mailjet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,599 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Mailjet Contact Management",
+    "description": "Contact Management in Mailjet is built to help you connect your contacts to mailjet making them easier to use in campaigns.",
+    "version": "1.0",
+    "contact": {
+      "name": "Mailjet Support",
+      "url": "https://www.mailjet.com/feature/support/",
+      "email": "contact@mailjet.com"
+    }
+  },
+  "host": "api.mailjet.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.mailjet.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.mailjet.com/security-privacy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Business Management; Business Intelligence"
+    }
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v3/REST/contact": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create a Contact",
+        "operationId": "CreateAContact",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "Content-Type",
+            "x-ms-summary": "Content-Type"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "IsExcludedFromCampaigns": {
+                  "type": "string",
+                  "description": "IsExcludedFromCampaigns"
```

Comment:
@GeoffryAbiFarah All the description values of properties should be free from grammatical errors.